README.md

```diff
@@ -123,27 +176,49 @@ get_built_package() {
 # export DATE_NUM="$(date -u +%Y%m%d%H%M%S)"
 # export DATE_STR="$(date -u +%Y%m%d_%H%M%S)"
 #
-# Or lock to date/time of the last commit on git, as Travis CI config does
-# (see .travis/common.sh)
+# Or lock to date/time of the last commit on git, as it's done in the CI:
 #
-export DATE_TS="$(git log --format=%ct -n1)"
+DATE_TS="$(git log --format=%ct -n1)"
 export DATE_NUM="$(date --date=@${DATE_TS} -u +%Y%m%d%H%M%S)"
 export DATE_STR="$(date --date=@${DATE_TS} -u +%Y%m%d_%H%M%S)"
+```
+
+### Preparing and building the package
+
+After getting all prerequisites and setting the required variables, it
+is recommended to prepare the recipe before building, as it gives you
+the advantages described on [the `conda-build-prepare`'s
+GitHub page](https://github.com/litex-hub/conda-build-prepare).
+The tool is also used within the CI, which makes the locally built
+package much more similar to the one built by the CI workflow.
+
+The `WORKDIR` variable is the name of the directory that will be
```

Comment:
I don't like the name `WORKDIR` -- I would prefer something more descriptive like `FINALIZE_PACKAGE_OUTPUTDIR` or something along those lines?

Reply:
I've changed it to `PREPARED_RECIPE_OUTPUTDIR` since it's a path of the new directory that will contain output after preparing the recipe. Of course I'm open for modifications if you think it should stress the fact that it's finalized or something else. I think no name is fully accurate apart from the longish and less descriptive `CONDA_BUILD_PREPARE_OUTPUTDIR. Perhaps `CBP_OUTPUTDIR` could be used but it's still less descriptive.